.github/workflows/publish-release.yml

```
@@ -26,6 +26,8 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
+      - run: yarn --frozen-lockfile
+      - run: yarn allow-scripts
       - uses: MetaMask/action-publish-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Probably best to install deps as late as possible, to mitigate the possibility of a bad postinstall script corrupting something.

```suggestion
      - uses: MetaMask/action-publish-release@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      - run: yarn --frozen-lockfile
      - run: yarn allow-scripts
```